content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,172 @@
+
+- **Added** Listener Access Logs when `ResponseFlag` from Envoy is set.
+  ([Issue #26851](https://github.com/istio/istio/issues/26851))
+
+- **Added** support for `OpenCensusAgent` formatted trace export with configurable trace context headers.
+
+- **Added** Proxy config to control Envoy native stats generation.
+  ([Issue #26546](https://github.com/istio/istio/issues/26546))
+
+- **Added** Istio Wasm Extension Grafana Dashboard.
+  ([Issue #25843](https://github.com/istio/istio/issues/25843))
+
+- **Fixed** client side metric  missing destination workload labels for out-of-mesh destination or request that fails to reach destination.
+  ([Issue #20538](https://github.com/istio/istio/issues/20538))
+
+- **Fixed** interpretation of `$(HOST_IP)` in Zipkin and Datadog tracer address.
+  ([Issue #27911](https://github.com/istio/istio/issues/27911))
+
+- **Removed** all Mixer-related features and functionality. This is a scheduled
+removal of a deprecated Istio services and deployments, as well as
+Mixer-focused CRDs and component and related functionality.
+  ([Issue #25333](https://github.com/istio/istio/issues/25333)),([Issue #24300](https://github.com/istio/istio/issues/24300))
+
+## Installation
+
+- **Improved** multi-network configuration so that labeling a service with `topology.istio.io/network=network-name` can
+configure cross-network gateways without using [mesh networks](/docs/reference/config/istio.mesh.v1alpha1/#MeshNetworks).
+
+- **Improved** sidecar injection to not modify the pod `securityPolicy.fsGroup` which could conflict with existing settings and secret mounts.
+ This option is enabled automatically on Kubernetes 1.19+ and is not supported on older versions.
+  ([Issue #26882](https://github.com/istio/istio/issues/26882))
+
+- **Improved** Generated operator manifests for use with `kustomize` are available in the [manifests]({{</* github_tree */>}}/manifests/charts/istio-operator/files) directory.
+  ([Issue #27139](https://github.com/istio/istio/issues/27139))
+
+- **Updated** install script to bypass GitHub API Rate Limiting.
+
+- **Added** port `15012` to the default list of ports for the `istio-ingressgateway` Service.
+  ([Issue #25933](https://github.com/istio/istio/issues/25933))
+
+- **Added** Istio 1.8 supports Kubernetes versions 1.17 to 1.19.
+  ([Issue #25793](https://github.com/istio/istio/issues/25793))
+
+- **Added** The network for a Pod can be specified via the label `topology.istio.io/network`. This overrides the setting for the cluster's installation values (`values.globalnetwork`). If the label isn't set, it is injected based on the global value for the cluster.
+  ([Issue #25500](https://github.com/istio/istio/issues/25500))
+
+- **Deprecated** installation flags `values.global.meshExpansion.enabled` in favor of user-managed config and `values.gateways.istio-ingressgateway.meshExpansionPorts` in favor of `components.ingressGateways[name=istio-ingressgateway].k8s.service.ports`
```

Comment:
@tbarrella Is this in 1.8 or 1.9? The linked issue (https://github.com/istio/istio/issues/25933) makes it sound like it will be in 1.9. If not, should we close that issue and track the rest of the progress in another?

Reply:
Hello, the release note and upgrade note are for 1.8; 1.9+ is for removing the flag completely